airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/oauth.py

```diff
@@ -22,7 +22,7 @@ def __init__(
         token_refresh_endpoint: str,
         client_id: str,
         client_secret: str,
-        refresh_token: str,
```

Comment:
why is the type annotation removed?

Reply:
That was an oversight, I will add it back